app/internal/page/data.go

```diff
@@ -130,6 +137,40 @@ func (a Address) String() string {
 	return strings.Join(parts, ", ")
 }
 
+func TransformAddressDetailsToAddress(ad ordnance_survey.AddressDetails) Address {
```

Comment:
I was on the fence about where Address should live. Initially, I'd moved it into the `ordnance_survey` but as we were treating it as part of our the app/form data models it felt like it should be part of `page` as changes to the package in the future could have side effects (although admittedly I guess `Address` is quite a known static entity).

This may just be me getting used to how we work with types in different packages in Go but happy to move this over as it will simplify things quite a bit.